sql/core/src/test/resources/sql-tests/inputs/udf/udf-except-all.sql

```diff
@@ -0,0 +1,162 @@
+-- This test file was converted from except-all.sql.
+
+CREATE TEMPORARY VIEW tab1 AS SELECT * FROM VALUES
+    (0), (1), (2), (2), (2), (2), (3), (null), (null) AS tab1(c1);
+CREATE TEMPORARY VIEW tab2 AS SELECT * FROM VALUES
+    (1), (2), (2), (3), (5), (5), (null) AS tab2(c1);
+CREATE TEMPORARY VIEW tab3 AS SELECT * FROM VALUES
+    (1, 2), 
+    (1, 2),
+    (1, 3),
+    (2, 3),
+    (2, 2)
+    AS tab3(k, v);
+CREATE TEMPORARY VIEW tab4 AS SELECT * FROM VALUES
+    (1, 2), 
+    (2, 3),
+    (2, 2),
+    (2, 2),
+    (2, 20)
+    AS tab4(k, v);
+
+-- Basic EXCEPT ALL
+SELECT udf(c1) FROM tab1
+EXCEPT ALL
+SELECT udf(c1) FROM tab2;
+
+-- MINUS ALL (synonym for EXCEPT)
+SELECT udf(c1) FROM tab1
+MINUS ALL
+SELECT udf(c1) FROM tab2;
+
+-- EXCEPT ALL same table in both branches
+SELECT udf(c1) FROM tab1
+EXCEPT ALL
+SELECT udf(c1) FROM tab2 WHERE udf(c1) IS NOT NULL;
+
+-- Empty left relation
+SELECT udf(c1) FROM tab1 WHERE udf(c1) > 5
+EXCEPT ALL
+SELECT udf(c1) FROM tab2;
+
+-- Empty right relation
+SELECT udf(c1) FROM tab1
+EXCEPT ALL
+SELECT udf(c1) FROM tab2 WHERE udf(c1 > udf(6));
```

Comment:
Is it intentionally to do `udf(6)`? Not `udf(c1) > 6`?

Reply:
Yes, I am trying a different combination of `udf`s.

Reply:
Yea, it's kind of a bit random but let's test different cases while we're here..